torchao/prototype/quantized_training/int8.py

```diff
@@ -0,0 +1,247 @@
+from typing import Any, Optional, Tuple
+
+import torch
+from torch import Tensor, nn
+from torch.utils._python_dispatch import return_and_correct_aliasing
+
+from torchao.dtypes.utils import _dispatch__torch_dispatch__, _dispatch__torch_function__, _implements
+
+
+aten = torch.ops.aten
+c10d_functional = torch.ops.c10d_functional
+_c10d_functional = torch.ops._c10d_functional
+
+
+# the main difference of this tensor subclass from AffineQuantizedTensor:
```

Comment:
perhaps @jerryzh168 has some feedback on similarities to AffineQuantizedTensor - granted i'd be fine if we punt on unification to another PR

Reply:
yeah I feel we should just enable this in `AffineQuantizedTensor